README.md

```diff
@@ -1,6 +1,6 @@
-# Coreto KPI collection SDK
+# Coreto RCS SDK
 
-An SDK used to for the collection and validation of the KPIs used in the Coreto Trust and Performance Ledger
+An Laravel SDK used for the collection and validation of the KPIs used in the Coreto DRS
```

Comment:
Is this only Laravel specific, or can it be used in other PHP frameworks as well?